crowbar_framework/app/models/service_object.rb

```diff
@@ -1193,6 +1205,17 @@ def apply_role(role, inst, in_queue)
           tmprole = RoleObject.find_role_by_name "#{role_name}_remove"
           unless tmprole.nil?
             elem_remove = tmprole.name
+            # save remove intention in #{@bc_name}-databag
+            databag["deployment"][@bc_name]["elements"][elem_remove] ||= []
+
+            old_nodes.each do |old_node|
+              @logger.debug "saving #{elem_remove} intention for #{old_node}"
+              unless databag["deployment"][@bc_name]["elements"][elem_remove].include? old_node
+                databag["deployment"][@bc_name]["elements"][elem_remove] << old_node
+              end
+            end
+
+            databag.save
```

Comment:
@bkutil will saving like this trigger a bump in the revision, which would result in making the proposal not appear as applied for the latest revision?